app/views/hq/users/show.html.erb

```diff
@@ -18,7 +18,7 @@
   <%= pluralize(@user.inactive_sponsors.count.to_s, 'Inactive Sponsor') %>
   <% if @user.sponsors.count > 0 %>
     <div class="inactive_sponsors_index">
-      <%= render partial: 'hq/sponsors/sponsors.html.haml', locals: {sponsors: @user.inactive_sponsors} %>
+      <%= render partial: 'hq/sponsors/sponsors.html.haml', locals: {sponsors: @user.inactive_sponsors.paginate(:page => params[:page])} %>
```

Comment:
This should probably be discussed; do we want both copies of the view partial paginating to the same place?  I would enter a task to uncouple these under osra327, whether that can be done directly or if it means separating the partial into two partials, one for Sponsor and one for User.  Another option would be to not paginate these as each user probably wont have an absurd amount of Sponsors